docs/dev/building-on-zksync/contracts/contract-development.md

```diff
@@ -0,0 +1,41 @@
+# Smart contract development
+
+zkSync Era allows developers to build projects using the same programming languages and tools as required by Ethereum.
+We provide a customized and optimized compiler toolchain for development on zkSync. For more information, see
+[the toolchain documentation](../../compiler-toolchain/overview.md).
+
+## EVM compatibility
+
+zkSync Era supports almost every smart contract written for EVM and holds all key security invariants so
```

Comment:
I’d suggest “satisfies” rather than “holds.”  (In this context, “holds” is used of properties rather than invariants, e.g. “A representation invariant is a property that holds of all values of…” (https://www.cs.princeton.edu/courses/archive/fall21/cos326/lec/13-01-representation-invariants.pdf)

Reply:
Please do in your PR.

Reply:
191a58b3